Kickstarter-iOS/Features/PledgedProjectsOverview/CardView/PPOAddressSummary.swift

```diff
@@ -0,0 +1,46 @@
+import Foundation
+import SwiftUI
+
+struct PPOAddressSummary: View {
+  let address: String
+
+  var body: some View {
+    HStack(alignment: .firstTextBaseline) {
+      // TODO: Localize
+      Text("Shipping address")
+        .lineLimit(nil)
+        .font(Font(Constants.labelFont))
+        .foregroundStyle(Color(Constants.labelColor))
+        .frame(width: Constants.labelWidth, alignment: .leading)
+        .multilineTextAlignment(.leading)
+
+      Text(self.address)
+        .lineLimit(nil)
+        .multilineTextAlignment(.leading)
+        .font(Font(Constants.addressFont))
+        .foregroundStyle(Color(Constants.addressColor))
+        .frame(maxWidth: .infinity, alignment: .leading)
+    }
+    .frame(maxWidth: .infinity)
+  }
+
+  private enum Constants {
```

Comment:
Is there any reason we are using enum here instead of struct?

Reply:
`enum`s without cases cannot be instantiated, so they are good for use as namespaces

Reply:
Okay, sounds good.